whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapService.java

```diff
@@ -245,6 +250,38 @@ public Response help(@Context final HttpServletRequest request) {
                 .build();
     }
 
+    @GET
+    @Produces({MediaType.APPLICATION_JSON, CONTENT_TYPE_RDAP_JSON})
+    @Path("/{objectType}/rirSearch1/{relation}/{key:.*}")
+    public Response relationSearch(
+            @Context final HttpServletRequest request,
+            @PathParam("objectType") RdapRequestType requestType,
+            @PathParam("relation") RelationType relationType,
+            @PathParam("key") final String key,
+            @QueryParam("status") String status) {
+
+        //TODO: [MH] Status is being ignored until administrative resources are included in RDAP. If status is not
+        // given or status is inactive...include administrative resources in the output. However, if status is active
+        // return just non administrative resources, as we are doing now.
+        if (status != null && (relationType.equals(RelationType.DOWN) || relationType.equals(RelationType.BOTTOM))){
+            throw new RdapException("501 Not Implemented", "Status is not implement in down and bottom relation", HttpStatus.NOT_IMPLEMENTED_501);
+        }
```

Comment:
should we just check if status is inactive and UP and DOWN? as by default we do return active status , also add a notification status is ignored rather than sending error?

Reply:
I added in the README. So I think that it should be enough. Community should know that for now we only consider active status

Relation Searches doesn't consider the "status" query parameter
-----------------------------------------------------------------
This is related to the previous point. An object with "administrative" status is never returned.
Therefore, only "active" (non-administrative) objects are taking into account when using relation searches.

Reply:
Yes but if a user is sending an active status and we are supporting it , we should return the object. It is inactive we are not supporting. No?

Reply:
In addition
The task says: Return "501" if status is combined with "down" or "bottom" relation

Reply:
yes we should return 501 only if status is inactive